lib/install/web.rb

```diff
@@ -27,6 +27,8 @@ def apply_template!
       gem_group :development, :test do
         if ARGV.include?("--suspenders-main")
           gem "suspenders", github: "thoughtbot/suspenders", branch: "main"
+        elsif ARGV.include?("--suspenders-local") && ENV["SUSPENDERS_LOCAL_PATH"]
+          gem "suspenders", path: ENV.fetch("SUSPENDERS_LOCAL_PATH")
```

Comment:
This feels a little redundant. Would this be better?

  ```suggestion
        elsif ENV["SUSPENDERS_LOCAL_PATH"].present?
          gem "suspenders", path: ENV.fetch("SUSPENDERS_LOCAL_PATH")
```

Then, we could omit `--suspenders-local` altogether.

 ```
 SUSPENDERS_LOCAL_PATH=/path/to/suspenders \
  rails new app_name \
  --skip-rubocop \
  --skip-test \
  -d=postgresql \
  -m=https://raw.githubusercontent.com/thoughtbot/suspenders/main/lib/install/web.rb
 ```

Reply:
Could we just let it use any path you give it to the template? Do you know if we have the value of `-m` at this point?

I'm thinking you could do something like this then:

```
value_of_m = "https://raw.githubusercontent.com/thoughtbot/suspenders/main/lib/install/web.rb"
uri = URI.parse(value_of_m)

if uri.kind_of? URI::Generic
  gem "suspenders", path: File.expand_path(value_of_m)
end
```

[This is just playing around with a URL parsing example](https://stackoverflow.com/a/13041565)